x-pack/plugins/ml/public/application/jobs/new_job/common/job_creator/categorization_job_creator.ts

```diff
@@ -204,4 +184,43 @@ export class CategorizationJobCreator extends JobCreator {
       }
     }
   }
+
+  // override the setter and getter for the per-partition toggle
+  // so we can remove the partition field in the wizard when
+  // per-partition categorization is disabled.
+  public get perPartitionCategorization() {
+    return this._job_config.analysis_config.per_partition_categorization?.enabled === true;
+  }
+
+  public set perPartitionCategorization(enabled: boolean) {
+    this._initPerPartitionCategorization();
+    this._job_config.analysis_config.per_partition_categorization!.enabled = enabled;
+    if (enabled === false) {
+      this.categorizationPerPartitionField = null;
+    }
+  }
+
+  public cloneFromExistingJob(job: Job, datafeed: Datafeed) {
+    this._overrideConfigs(job, datafeed);
+    this.createdBy = CREATED_BY_LABEL.CATEGORIZATION;
+    const detectors = getRichDetectors(job, datafeed, this.additionalFields, false);
+
+    const dtr = detectors[0];
+    if (dtr !== undefined && dtr.agg !== null && dtr.field !== null) {
+      const detectorType =
+        dtr.agg.id === ML_JOB_AGGREGATION.COUNT
+          ? ML_JOB_AGGREGATION.COUNT
+          : ML_JOB_AGGREGATION.RARE;
+
+      this.setDetectorType(detectorType);
+      if (dtr.partitionField !== null) {
+        this.categorizationPerPartitionField = dtr.partitionField.id;
+      }
+
+      const bs = job.analysis_config.bucket_span;
```

Comment:
I am finding the same issue as in the functional tests, with the bucket span of the cloned job being reset to 15m.

Reply:
Retested and confirm fixed by https://github.com/elastic/kibana/pull/86635/commits/8184f10b23fe93feca7c68804d546374fd27d32c